Cargo.toml

```diff
@@ -76,6 +76,12 @@ overflow-checks = true
 # "release" although they incur some performance penalty.
 overflow-checks = true
 
+[profile.release-with-debug]
```

Comment:
This ensures we actually build with debug in this repo (as crate-level overrides don't work)